src/types/biztechEvent.ts

```diff
@@ -0,0 +1,14 @@
+// type definition for a BiztechEvent
+export type BiztechEvent = {
```

Comment:
this is missing fields. reference https://github.com/ubc-biztech/bt-web-v2/blob/afd8e3d7662c0a378e159c754770c82be1584ed2/src/types.ts

Reply:
bruh i realized my comments from before never went through

Reply:
I will update these fields thanks allan 👍